src/sql/workbench/contrib/executionPlan/browser/executionPlanFileView.ts

```diff
@@ -28,7 +28,7 @@ export class ExecutionPlanFileView extends Disposable {
 	private _planCache: Map<string, azdata.executionPlan.ExecutionPlanGraph[]> = new Map();
 
 	constructor(
-		private _queryResultsView: QueryResultsView,
+		private _queryResultsView: QueryResultsView | undefined,
```

Comment:
Did you make sure everything below is handling undefined correctly?

Reply:
Yup, I have checked that case